api/src/main/java/org/commonjava/indy/conf/InternalFeatureConfig.java

```diff
@@ -33,6 +33,8 @@
 
     private Boolean storeValidation;
 
+    private String affectedGroupsExcludeFilter;
```

Comment:
I'd probably put this in the main IndyConfig. InternalFeatureConfig is more about managing feature flags (turning features off when they're not quite done, etc), not things we'd really expose for configuring user-facing features themselves.

Reply:
Fair enough. I am updating it.